### ISSUE TITLE
No tabs appearing in the Automatic Scene Switcher window

### ISSUE THREAD

**Issue by risharde** [WarmUpTill/SceneSwitcher#22]
Hi, I downloaded and installed Automatic Scene Switcher to my OBS Studio 25.0.4 install. Opened the Automatic Scene Switcher but I see no tabs at the top of the window. I was intending to use this for timed switching between scenes. I updated OBS Studio to 25.0.8 but I do not see it either.

This is on OBS Studio 64-bit version using the latest Scene Switcher (64-bit) which has been copied to the OBS Studio 64bit directory.

VC Redist 2017 was also reinstalled. This is on a Windows 64-bit pro machine.

Please advise? Desperate to get this working properly.

**Comment by WarmUpTill** [comment]
Hi,
are you sure you put the dll into the correct directory?
Can you send me a log file?

**Comment by risharde** [comment]
Thanks for the quick response, yes I am sure I put it in the 64bit folder
of OBS Studio such that even OBS Studio loads it in and I can select it in
the menu to open the switcher window.

Unfortunately already away from keyboard but I can try to get those logs to
you later today, so sorry

**Comment by risharde** [comment]
@WarmUpTill here is the logs (I don't see the dll listed in here but I will provide a screenshot as well to show that it does indeed load on OBS Studio)

`
11:16:45.096: CoInitializeEx succeeded: 0x00000001
11:16:45.096: CPU Name: AMD A8-7600 Radeon R7, 10 Compute Cores 4C+6G  
11:16:45.096: CPU Speed: 3094MHz
11:16:45.096: Physical Cores: 2, Logical Cores: 4
11:16:45.096: Physical Memory: 7110MB Total, 2810MB Free
11:16:45.096: Windows Version: 10.0 Build 18363 (release: 1909; revision: 778; 64-bit)
11:16:45.096: Running as administrator: false
11:16:45.096: Aero is Enabled (Aero is always on for windows 8 and above)
11:16:45.097: Windows 10 Gaming Features:
11:16:45.097: 	Game DVR: On
11:16:45.102: Sec. Software Status:
11:16:45.104: 	AVG Antivirus: enabled (AV)
11:16:45.105: 	Windows Defender Antivirus: disabled (AV)
11:16:45.105: 	Windows Firewall: enabled (FW)
11:16:45.106: Current Date/Time: 2020-05-03, 11:16:45
11:16:45.106: Browser Hardware Acceleration: true
11:16:45.106: Portable mode: false
11:16:45.590: OBS 25.0.8 (64-bit, windows)
11:16:45.590: ---------------------------------
11:16:45.636: ---------------------------------
11:16:45.636: audio settings reset:
11:16:45.636: 	samples per sec: 44100
11:16:45.636: 	speakers:        2
11:16:45.652: ---------------------------------
11:16:45.652: Initializing D3D11...
11:16:45.652: Available Video Adapters: 
11:16:45.658: 	Adapter 0: AMD Radeon(TM) R7 Graphics
11:16:45.658: 	  Dedicated VRAM: 1039400960
11:16:45.658: 	  Shared VRAM:    3728066560
11:16:45.659: 	  PCI ID:         1002:1313
11:16:45.659: 	  Driver Version: 26.20.12028.2
11:16:45.660: 	  output 0: pos={0, 0}, size={1360, 768}, attached=true, refresh=60, name=LG TV
11:16:45.660: 	  output 1: pos={1360, 0}, size={1440, 900}, attached=true, refresh=60, name=HW191
11:16:45.660: 	  output 2: pos={2800, 0}, size={1024, 768}, attached=true, refresh=120, name=Hanns.G HW191
11:16:45.665: Loading up D3D11 on adapter AMD Radeon(TM) R7 Graphics (0)
11:16:45.694: D3D11 loaded successfully, feature level used: b000
11:16:45.694: DXGI increase maximum frame latency success
11:16:45.695: D3D11 GPU priority setup failed (not admin?)
11:16:46.522: ---------------------------------
11:16:46.522: video settings reset:
11:16:46.522: 	base resolution:   1360x768
11:16:46.522: 	output resolution: 1280x720
11:16:46.522: 	downscale filter:  Bicubic
11:16:46.522: 	fps:               30/1
11:16:46.522: 	format:            NV12
11:16:46.522: 	YUV mode:          601/Partial
11:16:46.522: NV12 texture support enabled
11:16:46.524: Audio monitoring device:
11:16:46.524: 	name: Default
11:16:46.524: 	id: default
11:16:46.525: ---------------------------------
11:16:46.600: [CoreAudio encoder]: CoreAudio AAC encoder not installed on the system or couldn't be loaded
11:16:46.627: Failed to load 'en-US' text for module: 'decklink-ouput-ui.dll'
11:16:46.941: [AMF] The detected AMF runtime is too old, please update your drivers.
11:16:46.941: [AMF] AMF Runtime is outdated.
11:16:47.048: [AMF] AMF Test failed due to one or more errors.
11:16:47.048: Failed to initialize module 'enc-amf.dll'
11:16:47.741: [obs-browser]: Version 2.8.6
11:16:48.050: [obs-ndi] hello ! (version 4.7.1)
11:16:48.050: [obs-ndi] Trying 'e:\Program Files\NewTek\NDI 4 Runtime\v4'
11:16:48.050: [obs-ndi] Found NDI library at 'E:/Program Files/NewTek/NDI 4 Runtime/v4/Processing.NDI.Lib.x64.dll'
11:16:48.094: [obs-ndi] NDI runtime loaded successfully
11:16:48.100: [obs-ndi] NDI library initialized successfully (NDI SDK WIN64 15:59:50 Feb  4 2020 4.1.6.0)
11:16:49.864: VLC found, VLC video source enabled
11:16:49.992: A DeckLink iterator could not be created.  The DeckLink drivers may not be installed
11:16:49.994: No blackmagic support
11:16:50.083: ---------------------------------
11:16:50.083:   Loaded Modules:
11:16:50.083:     win-wasapi.dll
11:16:50.083:     win-mf.dll
11:16:50.083:     win-ivcam.dll
11:16:50.083:     win-dshow.dll
11:16:50.083:     win-decklink.dll
11:16:50.083:     win-capture.dll
11:16:50.083:     vlc-video.dll
11:16:50.083:     text-freetype2.dll
11:16:50.083:     rtmp-services.dll
11:16:50.083:     obs-x264.dll
11:16:50.083:     obs-vst.dll
11:16:50.083:     obs-transitions.dll
11:16:50.083:     obs-text.dll
11:16:50.083:     obs-qsv11.dll
11:16:50.083:     obs-outputs.dll
11:16:50.083:     obs-ndi.dll
11:16:50.083:     obs-filters.dll
11:16:50.083:     obs-ffmpeg.dll
11:16:50.083:     obs-browser.dll
11:16:50.083:     image-source.dll
11:16:50.083:     frontend-tools.dll
11:16:50.083:     enc-amf.dll
11:16:50.083:     decklink-ouput-ui.dll
11:16:50.083:     coreaudio-encoder.dll
11:16:50.083: ---------------------------------
11:16:50.083: ==== Startup complete ===============================================
11:16:50.095: All scene data cleared
11:16:50.095: ------------------------------------------------
11:16:50.172: WASAPI: Device 'Speakers (6- Yeti Stereo Microphone)' [48000 Hz] initialized
11:16:50.172: [Loaded global audio device]: 'Desktop Audio'
11:16:50.300: WASAPI: Device 'Microphone (HD Pro Webcam C920)' [16000 Hz] initialized
11:16:50.333: adding 46 milliseconds of audio buffering, total audio buffering is now 46 milliseconds (source: Audio - LOGI 2)
11:16:50.333: 
11:16:50.354: WASAPI: Device 'Microphone (6- Yeti Stereo Microphone)' [44100 Hz] initialized
11:16:50.646: [rtmp-services plugin] Successfully updated file 'services.json' (version 132)
11:16:50.648: [rtmp-services plugin] Successfully updated package (version 132)
11:16:50.862: Switched to scene 'WOLFC - LIVE SOON'
11:16:50.862: ------------------------------------------------
11:16:50.862: Loaded scenes:
11:16:50.862: - scene 'WOLFC - TEST BROADCAST':
11:16:50.862:     - source: 'WHITE' (color_source)
11:16:50.862:     - source: 'WOLFC LOGO' (image_source)
11:16:50.862:     - source: 'Divider' (image_source)
11:16:50.863:     - source: 'PREPARING TO GO LIVE! 2 2' (text_gdiplus)
11:16:50.863: - scene 'WOLFC - LIVE SOON':
11:16:50.863:     - source: 'WHITE' (color_source)
11:16:50.863:     - source: 'WOLFC LOGO' (image_source)
11:16:50.863:     - source: 'WORD OF LIFE FAMILY CHURCH' (text_gdiplus)
11:16:50.863:     - source: 'PREPARING TO GO LIVE!' (text_gdiplus)
11:16:50.863: - scene 'WOLFC - BE RIGHT BACK':
11:16:50.863:     - source: 'WHITE' (color_source)
11:16:50.863:     - source: 'WOLFC LOGO' (image_source)
11:16:50.863:     - source: 'PREPARING TO GO LIVE! 2' (text_gdiplus)
11:16:50.863:     - source: 'Divider' (image_source)
11:16:50.863: - scene 'Scene - Camera 2 - MOM CAMERA':
11:16:50.863:     - source: 'LOGI 2' (dshow_input)
11:16:50.863:         - filter: 'Color Correction' (color_filter)
11:16:50.863:     - source: 'Image 2' (image_source)
11:16:50.863:     - source: 'Audio - LOGI 2' (wasapi_input_capture)
11:16:50.863: - scene 'Scene - Camera 1 - MY CAMERA':
11:16:50.863:     - source: 'LOGI1' (dshow_input)
11:16:50.863:         - filter: 'Color Correction' (color_filter)
11:16:50.863:     - source: 'Image' (image_source)
11:16:50.863:     - source: 'AUDIO - YETI' (wasapi_input_capture)
11:16:50.863:     - source: 'Pastor Risharde' (text_gdiplus_v2)
11:16:50.863:     - source: 'Pastor - Word of Love Family Church' (text_gdiplus_v2)
11:16:50.863: - scene 'Scene - Display 1 - POWERPOINT':
11:16:50.863:     - source: 'Display 1' (monitor_capture)
11:16:50.863:         - filter: 'Color Correction' (color_filter)
11:16:50.863:     - source: 'LOGI1' (dshow_input)
11:16:50.863:         - filter: 'Color Correction' (color_filter)
11:16:50.863:     - source: 'AUDIO - YETI' (wasapi_input_capture)
11:16:50.863: - scene 'Scene - Display 3 - Zoom':
11:16:50.863:     - source: 'WHITE BACKGROUND' (color_source_v2)
11:16:50.863:     - source: 'Display 3' (monitor_capture)
11:16:50.863:     - source: 'WOLFC LOGO' (image_source)
11:16:50.863:     - source: 'AUDIO - YETI' (wasapi_input_capture)
11:16:50.863:     - source: 'Daren Mohamed' (text_gdiplus_v2)
11:16:50.863:     - source: 'Leader - Prayer Ministry' (text_gdiplus_v2)
11:16:50.863: - scene 'Scene - Display 2 - OBS Studio - DO NOT CLICK':
11:16:50.863:     - source: 'Display 2' (monitor_capture)
11:16:50.863:     - source: 'AUDIO - YETI' (wasapi_input_capture)
11:16:50.864: ------------------------------------------------

`

**Comment by risharde** [comment]
Screenshot of the missing tabs:
https://ibb.co/yB9VrW1

**Comment by WarmUpTill** [comment]
The screenshot you provided is of the built in Scene Switcher of OBS.

Sorry to ask again, but as OBS does not seem to try to load the .dll and you can't see the plugin in the tools menu:

Are you sure you put the .dll into the correct folder?

Maybe you put the .dll into the "bin" folder instead of the "obs-plugins" one by accident?
C:\Program Files (x86)\obs-studio\\**bin**\\64bit
C:\Program Files (x86)\obs-studio\\**obs-plugins**\\64bit

**Comment by risharde** [comment]
@WarmUpTill Thank you! Indeed you were indeed right on track! I did put it in the bin folder. Due to my lack of experience (never using the scene switch on OBS as well, I did mistakenly think that it was your module). Indeed I am terribly sorry for using up your time. I did what you said and found the \obs-studio\obs-plugins\64bit directory, placed the dll in there, fired up scene switcher and it works!

Thank you once again!!!! Really appreciate it and once again, my sincerest apology for not doing it right in the first place
